gateway/app/schemas/quality_on_demand.py

```diff
@@ -0,0 +1,342 @@
+from __future__ import annotations
+
+from datetime import datetime
+from enum import Enum
+from typing import Annotated, Any, Dict, List, Optional
+from uuid import UUID
+
+from pydantic import AnyUrl, BaseModel, Field
+
+from app.schemas.device import Device
+
+
+SessionId = Annotated[UUID, Field(description="Session ID in UUID format")]
+
+Port = Annotated[int, Field(ge=0, le=65535, description="TCP or UDP port number.")]
+
+QosProfileName = Annotated[
+    str,
+    Field(
+        pattern=r"^[a-zA-Z0-9_.-]+$",
+        description="A unique name for identifying a specific QoS profile.\nThis may follow different formats depending on the API provider implementation.\nSome options addresses:\n  - A UUID style string\n  - Support for predefined profiles QOS_S, QOS_M, QOS_L, and QOS_E\n  - A searchable descriptive name\nThe set of QoS Profiles that an API provider is offering may be retrieved by means of the QoS Profile API (qos-profile) or agreed on onboarding time.\n",
+        min_length=3,
+        max_length=256,
+        examples=["voice"],
+    ),
+]
+
+NetworkAccessIdentifier = Annotated[
+    str,
+    Field(
+        description="A public identifier addressing a subscription in a mobile network. In 3GPP terminology, it corresponds to the GPSI formatted with the External Identifier ({Local Identifier}@{Domain Identifier}). Unlike the telephone number, the network access identifier is not subjected to portability ruling in force, and is individually managed by each operator.",
+        examples=["123456789@domain.com"],
+    ),
+]
+
+PhoneNumber = Annotated[
+    str,
+    Field(
+        pattern=r"^\+[1-9][0-9]{4,14}$",
+        description="A public identifier addressing a telephone subscription. In mobile networks it corresponds to the MSISDN (Mobile Station International Subscriber Directory Number). In order to be globally unique it has to be formatted in international format, according to E.164 standard, prefixed with '+'.",
+        examples=["+123456789"],
+    ),
+]
+
+
+class Range(BaseModel):
+    from_: Annotated[Port, Field(alias="from")]
+    to: Port
```

Comment:
```suggestion
class Range(BaseModel):
    model_config = ConfigDict(serialize_by_alias=True)

    from_: Annotated[Port, Field(alias="from")]
    to: Port
```

Adding `serialize_by_alias` that the model is serialized correctly even when `by_alias` is not set.